docker/phylo_dist/Dockerfile

```diff
@@ -12,8 +12,39 @@ RUN apt-get update && \
     g++ \
     cmake \
     git \
+    wget \
+    unzip \
     && rm -rf /var/lib/apt/lists/*
 
+# Install Miniconda for conda package management
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda.sh && \
+    bash /tmp/miniconda.sh -b -p /opt/conda && \
+    rm /tmp/miniconda.sh && \
+    /opt/conda/bin/conda clean -afy
+
+# Add conda to PATH and set CONDA_PREFIX for R scripts
+ENV PATH="/opt/conda/bin:${PATH}"
+ENV CONDA_PREFIX="/opt/conda"
+
+# Accept conda Terms of Service and configure channels
+RUN conda config --set channel_priority flexible && \
+    conda tos accept --override-channels --channel https://repo.anaconda.com/pkgs/main && \
+    conda tos accept --override-channels --channel https://repo.anaconda.com/pkgs/r
+
+# Install treePL via conda
+RUN conda install -c genomedk treepl -y && \
+    conda clean -afy
+
+# Download and compile PATHd8 from source
+# PATHd8 is a C program for phylogenetic dating, official site: www2.math.su.se/PATHd8/
+RUN wget https://www2.math.su.se/PATHd8/PATHd8.zip -O /tmp/PATHd8.zip && \
+    unzip /tmp/PATHd8.zip -d /opt/PATHd8_src && \
+    cd /opt/PATHd8_src && \
+    gcc PATHd8.c -O3 -lm -o PATHd8 && \
+    mv PATHd8 /usr/local/bin/ && \
+    cd / && \
+    rm -rf /opt/PATHd8_src /tmp/PATHd8.zip
```

Comment:
I'm glad this didn't cause any problems! Getting this to work was initially quite annoying...